modules/openapi-generator/src/main/resources/dart/libraries/dio/api.mustache

```diff
@@ -116,35 +119,15 @@ class {{classname}} {
       cancelToken: cancelToken,
       onSendProgress: onSendProgress,
       onReceiveProgress: onReceiveProgress,
-    );
-    {{#returnType}}
+    );{{^returnType}}
 
-    {{{returnType}}} _responseData;
-
-    try {
-{{#useBuiltValue}}{{>serialization/built_value/deserialize}}{{/useBuiltValue}}
-    } catch (error, stackTrace) {
-      throw DioError(
-        requestOptions: _response.requestOptions,
-        response: _response,
-        type: DioErrorType.other,
-        error: error,
-      )..stackTrace = stackTrace;
-    }
-
-    return Response<{{{returnType}}}>(
-      data: _responseData,
-      headers: _response.headers,
-      isRedirect: _response.isRedirect,
-      requestOptions: _response.requestOptions,
-      redirects: _response.redirects,
-      statusCode: _response.statusCode,
-      statusMessage: _response.statusMessage,
-      extra: _response.extra,
-    );{{/returnType}}{{^returnType}}
-    return _response;{{/returnType}}
```

Comment:
I would like the general try/catch/response handling to stay here, independent of built_value. If the `return_type` doesn't fit then we should fix that.